Doc/library/http.server.rst

```diff
@@ -462,6 +496,27 @@ following command runs an HTTP/1.1 conformant server::
 .. versionchanged:: 3.11
    Added the ``--protocol`` option.
 
+The server can also support TLS encryption. The options ``--tls-cert`` and
+``--tls-key`` allow specifying a TLS certificate chain and private key for
+secure HTTPS connections. For example, the following command runs the server with
```

Comment:
```suggestion
``--tls-key`` allow specifying a TLS certificate chain and a private key for
HTTPS connections. For example, the following command runs the server with
```

I think "secure HTTPS" is pleonasm but since the S in HTTPS is already for "secure". In addition, we don't necessarily want to say that the command-line setups the HTTPS server in a secure manner.